cmd/kubeadm/app/util/system/types_windows.go

```diff
@@ -38,7 +38,7 @@ var DefaultSysSpec = SysSpec{
 	Cgroups: []string{},
 	RuntimeSpec: RuntimeSpec{
 		DockerSpec: &DockerSpec{
-			Version:     []string{`18\.06\..*`}, //Requires [18.06] or later
+			Version:     []string{`18\.0[6,9]\..*`}, //Requires [18.09] or later
```

Comment:
let's remove this comment too, i guess.
because we now support two versions.

Reply:
I understand what you mean. Thank you for your quick reply